EventFilter/CTPPSRawToDigi/interface/DiamondVFATFrame.h

```diff
@@ -0,0 +1,60 @@
+/**********************************************************
+*
+* Seyed Mohsen Etesami (setesami@cern.ch)    
+*
+**********************************************************/
+
+#ifndef EventFilter_CTPPSRawToDigi_DiamondVFATFrame
+#define EventFilter_CTPPSRawToDigi_DiamondVFATFrame
+
+#include <vector>
+#include <cstddef>
+#include <stdint.h>
+
+#include "EventFilter/CTPPSRawToDigi/interface/VFATFrame.h" 
+
+
+/** 
+ * This class intended to handle the timing infromation of diamond VFAT frame
+**/
+class DiamondVFATFrame : public VFATFrame
+{
+  
+  public:
+    DiamondVFATFrame(const word* _data = NULL)
```

Comment:
No name should have a leading underbar. Maybe change `_data` to `inputData`, but it's not essential.